services/ui-src/src/components/tables/EntityRow.tsx

```diff
@@ -112,7 +113,9 @@ export const EntityRow = ({
             variant="outline"
             disabled={entityStatus === "disabled"}
           >
-            {verbiage.enterEntityDetailsButtonText}
+            {report?.status === ReportStatus.SUBMITTED
```

Comment:
should this also apply to the Approved status?

Reply:
good point, I'll add that case! thank you 💯